OrbitGl/SamplingReportDataView.cpp

```diff
@@ -138,14 +137,15 @@ void SamplingReportDataView::DoSort() {
   std::sort(indices_.begin(), indices_.end(), combined_sorter);
 }
 
-std::vector<const FunctionInfo*> SamplingReportDataView::GetFunctionsFromIndices(
+absl::flat_hash_set<const FunctionInfo*> SamplingReportDataView::GetFunctionsFromIndices(
```

Comment:
Is there a reason for changing this, other than simplifying the code? From what I remember, the rationale for having a vector was returning the functions in the same order as the `indices`.

Reply:
It was already a std::set internally, which was then converted into a vector. That was unnecessarily complicated.

Reply:
You're right, forget about this.